### PR TITLE
✨  Update documentation

### PR DESCRIPTION
**Pull request by azeemshaikh38**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Updates existing documentation.

### CODE REVIEW

File: roundtripper/roundtripper.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Caching
```
Comment (inferno-chromium) [inline roundtripper/roundtripper.md:1]: usually these files can be named README.md (readme for a directory).

File: README.md [hunk 2]
```diff
@@ -247,112 +201,114 @@ export GITHUB_AUTH_TOKEN=<your access token>
...
+If you're only interested in seeing a list of projects with their Scorecard
+check results, we publish these results in a
+[BigQuery public dataset](https://cloud.google.com/bigquery/public-data).
 
-- ### Bugs and Feature Requests:
-  If you have what looks like a bug, or you would like to make a feature request, please use the [Github issue tracking system.](https://github.com/ossf/scorecard/issues)
-Before you file an issue, please search existing issues to see if your issue is already covered.
+This data is available in the public BigQuery dataset
+`openssf:scorecardcron.scorecard`. The latest results are available at
+`openssf:scorecardcron.scorecard_latest`.
```
Comment (inferno-chromium) [inline README.md:243]: Good to give example command to query results from data or lets say how to export them to csv or any other format.

**Review by inferno-chromium — APPROVED** [review]
This is awesome!

File: roundtripper/roundtripper.md (hunk 1 above)
Comment (azeemshaikh38) [reply roundtripper/roundtripper.md]: Done.

File: README.md (hunk 2 above)
Comment (azeemshaikh38) [reply README.md:258]: Done.

**Pull request merged**